cli/src/commands/deploy.ts

```diff
@@ -28,7 +31,7 @@ import {
 // * project associated deploys to associated app
 // * project associated and deploy fails shows meaningful message
 //
-// TODO: add --app-id and --env flags to bypass app association mechanism
+// TODO: add --env flag to bypass app association mechanism
```

Comment:
CR: remove the `--env` TODO too

Reply:
To be able to deploy to the non `default` app environment, different from env variables.